x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_utilities.tsx

```diff
@@ -6,10 +6,16 @@
 
 /* eslint-disable react/display-name */
 
-import { EuiCode } from '@elastic/eui';
+import { EuiCode, EuiToolTip, EuiPopover } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import styled from 'styled-components';
-import React, { memo } from 'react';
+import React, { memo, useState } from 'react';
+import { WithCopyToClipboard } from '../../../common/lib/clipboard/with_copy_to_clipboard';
+import { useColors } from '../use_colors';
+
+const COPY_TO_CLIPBOARD = i18n.translate('xpack.securitySolution.resolver.panel.copyToClipboard', {
```

Comment:
`i18n.translate` just returns a string. You shouldn't need to store a reference to it. Can you just place the expression inline in the react component instead?

Reply:
Yep